src/capitalsFirst.js

```diff
@@ -18,16 +18,20 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  let a = [];
-  let b = [];
+  let upperWords = [];
+  let lowerWords = [];
   str.split(' ').map((el) => {
-    if (el[0] === el[0].toUpperCase() && isNaN(el[0]) && el[0] !== '=') {
-      a.push(el);
-    } else if (isNaN(el[0]) && el[0] !== '=') {
-      b.push(el);
-    };
+    return isNaN(el[0]) && el[0] !== '='
```

Comment:
I believe `el[0] !== '=' ` is wrong. Words may start with other symbols which have to be skipped.